### PR TITLE
29946 - Content model updates

### PR DESCRIPTION
**Pull request by AlexW301**
#AB29946 - Updated kontent.ai content model and updated blog terminology to insights

### CODE REVIEW

File: contentstack-finserv/components/footer.tsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+import Link from 'next/link';
+import parse from 'html-react-parser';
+import { onEntryChange } from '../contentstack-sdk';
+import { getFooterRes } from '../helper';
+import Skeleton from 'react-loading-skeleton';
+import { FooterProps, Entry, Links } from "../typescript/layout";
+
+export default function Footer({ footer, entries }: {footer: FooterProps, entries: Entry}) {
+
+  const [getFooter, setFooter] = useState(footer);
```
Comment (mhak) [inline contentstack-finserv/components/footer.tsx:11]: do we need states for the footer? I wouldn't expect it to be reactive

**Review by mhak — CHANGES_REQUESTED** [review]

File: contentstack-finserv/components/footer.tsx (hunk 1 above)
Comment (AlexW301) [reply contentstack-finserv/components/footer.tsx:11]: I am not sure why that is in my pr, that footer is from the contentstack project. @zachary-horvath  ?

Comment (zachary-horvath) [reply contentstack-finserv/components/footer.tsx:11]: This is to be removed in the app router upgrade. State in the old app was used to update the data on the fly.

**Review by zachary-horvath — APPROVED** [review]
Approving this. File footer.tsx in contentstack-finserv/components does not exist in new app router branch.

**Comment by zachary-horvath** [comment]
Approved. See above comments.

**Pull request merged**